app/controllers/users_controller.rb

```diff
@@ -39,6 +39,7 @@ def create
 
   def show
     @user_tags = UserTag.new
+    @user_tags_list = UserTag.where(user_id: User.find(session[:user_id]))
```

Comment:
This code doesn't create a bug. However, the .find method returns an object not an id. In this case, you're setting user_id: to an object. I'm surprised this works. 

Also, session[:user_id] is already a user_id so you can just use UserTag.where(user_id: session[:user_id).

Reply:
@awong3321 Thanks for pointing this out! Could you take a look at the UserTags controller and see if there's anything else I need to bring over? In the UserTags controller I had an instance variable   @user = User.find(session[:user_id]). I'll change is to session[:user_id]

Reply:
Your UserTag Controller looks great.